src/services/TechRecordsService.ts

```diff
@@ -295,8 +293,8 @@ class TechRecordsService {
       if (isTrailerIdValid.error) {
         return Promise.reject({statusCode: 400, body: formatErrorMessage("TrailerId is invalid")});
       }
-      const trailerIdRecords: QueryOutput = await this.techRecordsDAO.getBySearchTerm(techRecord.trailerId, SEARCHCRITERIA.TRAILERID);
-      if (trailerIdRecords.Count! > 0) {
+      const trailerIdRecords: ITechRecordWrapper[] = await this.techRecordsDAO.getBySearchTerm(techRecord.trailerId, SEARCHCRITERIA.TRAILERID);
```

Comment:
👍  Much better as a ITechRecordWrapper array - You can get rid of QueryOutput from the imports :)

Reply:
This has now been removed, please resolve if happy with change. Thanks